compiler/shared/test/diff/Defunctionalize/SimpleFunc.mls

```diff
@@ -54,12 +54,16 @@ let b = foo(23, false)
 //│ }
 //│ Mono:
 //│ Bounds for Var(foo$3) not found during defunc.
-//│ ╔══[ERROR] Post-process failed to produce AST, defaulting to standard AST
```

Comment:
You should change:
```diff
-:e // TODO: Evaluate unused terms
+// TODO: Evaluate unused terms
```
to remove the stack trace from the test file.

Reply:
Oh, is this a `MonomorphError`, whose handling is special-cased below? Why the special case code?

Reply:
I don't think there's any particular reason other than to distinguish it from the other sections of the codebase. Would it be appropriate to change this to a `CodeGenError` and use the same error handling system?

Reply:
Added conditional that only outputs stack trace if the `:ge` flag is given